src/styles/blocks/page.scss

```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
+}
+
+p,
+h3 {
+  margin: 0;
+  padding: 0;
+}
+
+* {
```

Comment:
The 'box-sizing: border-box;' style should be applied globally, not just within the product card. Consider moving this to a separate, global SCSS file.